### ISSUE TITLE
get console output message

### ISSUE THREAD

**Issue by abc408880155** [SpigotMC/BungeeCord#3443]
### Feature description

in Bukkit, after send command to console ,could get console output message.

 public static @NotNull CommandSender createCommandSender(@NotNull Consumer<? super Component> feedback) {
        return server.createCommandSender(feedback);
    }

Is it possible to obtain command feedback messages from the BungeeCord console?

Or, there is a way, but I just haven't found it?

### Goal of the feature

obtain command feedback messages from the BungeeCord console

### Unfitting alternatives

sorry, dont know 

### Checking

- [X] This is not a question or plugin creation help request.
- [X] This is a **feature or improvement request**.
- [X] I have not read these checkboxes and therefore I just ticked them all.
- [X] I did not use this form to report a bug.

**Comment by Janmm14** [comment]
Its sad you don't take the time to read checkboxes.

Afaik bungeecord does not have any API to execute a command as console, therefore making this feature obsolete.

**Comment by Janmm14** [comment]
My bad, forgot that api exists

**Comment by Outfluencer** [comment]
I dont think you can do this with the console but you could implement the CommandSender yourself with all permissions and override the sendmessage method, so you could read the output their
